Sprintfinity3902/States/ItemPickup/HeartContainerPickup.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Sprintfinity3902.Link;
+using Sprintfinity3902.Interfaces;
+
+namespace Sprintfinity3902.Entities
+{
+    public class HeartContainerPickup : IPickup
+    {
+        public HeartContainerPickup()
+        { 
+
+        }
+
+        public bool Pickup(Player Link)
+        {
+            Link.MaxHealth += 2;
+            Link.LinkHealth += 2;
+            Link.heartChanged = true;
+            Sound.SoundLoader.Instance.GetSound(Sound.SoundLoader.Sounds.LOZ_Get_Heart).Play(Global.Var.VOLUME, Global.Var.PITCH, Global.Var.PAN);
```

Comment:
I am not sure if the Heart Container has a specific sound associated.

Reply:
Removed it!